java/client/src/main/java/glide/api/commands/StringBaseCommands.java

```diff
@@ -34,6 +34,26 @@ public interface StringBaseCommands {
      */
     CompletableFuture<String> get(String key);
 
+    /**
+     * Get the value associated with the given <code>key</code>. <code>GETDEL</code> is similar to
+     * {@see #get}, except for the fact that it also deletes the key on success (if and only if the
+     * value of the <code>key</code> is a string).
+     *
+     * @see <a href="https://redis.io/docs/latest/commands/getdel/">redis.io</a> for details.
+     * @param key The <code>key</code> to retrieve from the database.
+     * @return If <code>key</code> exists, returns the <code>value</code> of <code>key</code> as a
+     *     <code>String</code>. Otherwise, return <code>null</code>.
+     * @example
+     *     <pre>{@code
+     * String value = client.getdel("key").get();
+     * assert value.equals("value");
+     *
+     * String value = client.get("key").get();
```

Comment:
nit
```suggestion
     * String value = client.getdel("key").get();
```

Reply:
@Yury-Fridlyand Here i am trying to show that if they call get again, they will receive a null value

Reply:
oh wait but i can use getdel again here anyways